libs/designsystem/src/lib/components/item/label/label.component.scss

```diff
@@ -19,6 +20,12 @@
       margin-left: size('xs');
     }
   }
+
+  @include slotted('kirby-checkbox') {
+    ion-checkbox {
+      position: static;
+    }
+  }
```

Comment:
Consider moving this rule to `checkbox.component.scss` instead and using `host-context(kirby-label)` selector